packages/wp-dashboard/src/components/editorSettings/customFonts/index.js

```diff
@@ -209,8 +212,9 @@ function CustomFontsSettings({
       setToDelete(null);
       setShowDialog(false);
       setInputError('');
+      speak(__('Font deleted', 'web-stories'));
```

Comment:
Shouldn't this be within the `try` block? Otherwise this gets called even when deletion failed and an error snackbar is displayed

Reply:
Yes - your right